aws-athena/views/qc-vw_iasworld_change_in_low_value_sales.sql

```diff
@@ -0,0 +1,39 @@
+WITH sales_cte AS (
```

Comment:
**[Question, blocking]** Am I right that this view is basically identical to `qc-vw_iasworld_change_in_high_low_value_sales`? If so, can we consolidate them and move the tests for one view onto the other?

Reply:
I consolidated it, and thought I deleted this. You are right that it should be deleted.